executor/runcexecutor/executor.go

```diff
@@ -559,8 +658,16 @@ func handleSignals(ctx context.Context, runcProcess *procHandle, signals <-chan
 		case <-ctx.Done():
 			return nil
 		case sig := <-signals:
-			err := runcProcess.Process.Signal(sig)
-			if err != nil {
+			if sig == syscall.SIGKILL {
+				// never send SIGKILL directly to runc, it needs to go to the
+				// process in-container
+				if err := runcProcess.killer.Kill(ctx); err != nil {
+					bklog.G(ctx).Errorf("failed to kill the process: %+v", err)
```

Comment:
nit: I think this log will mostly be a duplicate of the log on line 480

Reply:
thanks fixed.